pan3d/dataset_viewer.py

```diff
@@ -249,56 +360,76 @@ def set_render_options(
         if self.builder.mesh is not None and self.builder.data_array is not None:
             self.apply_and_render()
 
-    async def plot_mesh(self) -> None:
+    def plot_mesh(self) -> None:
```

Comment:
no more async need? No need to make room for the network?

Reply:
Asynchronous parts of this function were moved to `run_as_async`.